modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PhoneResourceTest.java

```diff
@@ -258,6 +268,28 @@ private long _getListTypeId(String listTypeId) {
 		return listType.getListTypeId();
 	}
 
+	private void _testDeletePrimaryPhone() throws Exception {
+		Phone phone1 = randomPhone();
+
+		phone1.setPrimary(true);
+
+		phone1 = _addPhone(
+			phone1, Contact.class.getName(), _user.getContactId(),
+			ListTypeConstants.CONTACT_PHONE);
+
+		Assert.assertTrue(phone1.getPrimary());
+
+		Phone phone2 = testDeletePhone_addPhone();
+
+		Assert.assertFalse(phone2.getPrimary());
```

Comment:
I think this is not required if "false" is the default value during phone creation. If default value is "true" then line 274 is not required.